cmd/demo_scripts/create_qovery_demo.sh

```diff
@@ -6,6 +6,7 @@ QOVERY_API_URL=${QOVERY_API_URL:='https://api.qovery.com'}
 CLUSTER_NAME=$1
 ARCH=$2
 ORGANIZATION_ID=$3
+AGENT=$6
```

Comment:
minor: would have named it `USER_AGENT` (`AGENT` makes me think of qovery agent :smiley: )

Reply:
done